TASVideos/Pages/Wiki/Move.cshtml.cs

```diff
@@ -55,6 +55,17 @@ public async Task<IActionResult> OnPost()
 
 		var result = await wikiPages.Move(OriginalPageName, DestinationPageName);
 
+		// At a dummy commit to track the move
+		var page = (await wikiPages.Page(DestinationPageName))!;
+		await wikiPages.Add(new WikiCreateRequest
+		{
+			PageName = DestinationPageName,
+			Markup = page.Markup,
+			RevisionMessage = $"Page Moved from {OriginalPageName}",
```

Comment:
For cases when a page is moved several times, maybe also tell where the move is to?